packages/ethereum-wallets/src/lib/modal.ts

```
@@ -378,6 +378,77 @@ export function createTxModal({
   };
   return { showModal, hideModal, renderTxs };
 }
+export function createMessageModal({
+  onCancel,
+  title,
+  message
```

ESLint

[error] Insert `,`

(prettier/prettier)